client/pkg/transport/listener.go

```diff
@@ -415,23 +419,20 @@ func (info TLSInfo) baseConfig() (*tls.Config, error) {
 		cfg.CipherSuites = info.CipherSuites
 	}
 
+	var definedRestrictions int
```

Comment:
Is the following an upstream thing ?
I would have though that merging those flags to make more sense.

Reply:
I don't think so. This is what upstream looks like: https://github.com/etcd-io/etcd/blob/release-3.5/client/pkg/transport/listener.go#L422-L433.

@badouralix and I made this change after discussing here: https://github.com/DataDog/etcd/pull/5#discussion_r1781412282.